### ISSUE TITLE
Possible performance boost in `cellrank/tl/kernels/_velocity_schemes`

### ISSUE THREAD

**Issue by WeilerP** [theislab/cellrank#683]
## Description

In `cellrank/tl/kernels/_velocity_schemes:117`, `cellrank/tl/kernels/_velocity_schemes:131` and `cellrank/tl/kernels/_velocity_schemes:137`, `numba` throws a `NumbaPerformanceWarning`:

```pytb
NumbaPerformanceWarning: np.dot() is faster on contiguous arrays, called on (array(float64, 2d, C), array(float64, 1d, A))
```

See, for example, the testing stack trace of [this](https://github.com/theislab/cellrank/runs/3118375279?check_suite_focus=true) CI run.

**Comment by michalk8** [comment]
My quick test setup:
```python
import numpy as np
import cellrank as cr
from cellrank.tl.kernels._velocity_schemes import _predict_transition_probabilities_numpy
from time import time

ts = []
for i in range(101):
    x, W = np.random.normal(size=(100, 3000)), np.random.normal(size=(100, 3000))
    W = np.asfortranarray(W)
    t = time()
    _ = _predict_transition_probabilities_numpy(x, W)
    ts.append(time() - t)
np.mean(ts[1:]), np.var(ts[1:])
```
and I have added [here](https://github.com/theislab/cellrank/blob/master/cellrank/tl/kernels/_velocity_schemes.py#L103) `W = np.ascontiguousarray(W)`.
Without adding it, I've observed `(0.00344862699508667, 3.6004474065407516e-07)`.
With adding it, it was `(0.0036414241790771484, 5.000997375645965e-08)`.
However, I've observed the opposite happening (that requiring C-contiguous arrays makes it a bit faster).
Imho, this isn't enough for me to require it, so I'd close this issue.
Regarding the warning, locally I wasn't able to produce it (both in tests and in a notebook), but my test setup should be correct.
We can either silence them in the `VelocityScheme` class, only in tests or leave them be - I think it's fine to leave them there.
